### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by annabasenyuk**
Hi) Added the task solution, check it please)

### CODE REVIEW

File: src/herbivoresAndCarnivores.js [hunk 1]
```diff
@@ -1,15 +1,46 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    const COMPLEATLY_HEALTHY = 100;
```
Comment (polosanya) [inline src/herbivoresAndCarnivores.js:5]: Better move this variable outside of the constructor

File: src/herbivoresAndCarnivores.js [hunk 2]
```diff
@@ -1,15 +1,46 @@
...
 
 class Carnivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(health, name);
+  }
+
+  bite(prey) {
+    const BITE_OF_ANIMAL = 50;
+    const NOT_HEALTHY = 0;
```
Comment (polosanya) [inline src/herbivoresAndCarnivores.js:34]: Same here. Move all variables to the top

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost perfect!

File: src/herbivoresAndCarnivores.js (hunk 2 above)
Comment (annabasenyuk) [reply src/herbivoresAndCarnivores.js]: Thanks for your comment) I hope I understood correctly the comments)

**Review by polosanya — APPROVED** [review]
Well done! 👍

**Pull request closed without merging**